src/style.scss

```diff
@@ -38,6 +38,7 @@
 @import './inputs/input/input';
 @import './inputs/phoneInput/phoneInput';
 @import './inputs/prompt/prompt';
+@import './inputs/radio/radio';
```

Comment:
Please revert (see other comment; the `Radio` component is using JSS styling exclusively and we want to keep it that way).

Reply:
Looks good.  Resolving.